generators/base/generator-base-private.mjs

```diff
@@ -148,11 +147,8 @@ export default class PrivateBase extends Generator {
    * @param file
    */
   removeFile(file) {
-    file = this.destinationPath(file);
-    if (file && shelljs.test('-f', file)) {
-      this.log(`Removing the file - ${file}`);
-      shelljs.rm(file);
-    }
+    // TODO Should not update variable
+    file = deleteFile(this, file);
```

Comment:
```suggestion
    file = this.destinationPath(file);
    this.log(`Removing the file - ${file}`);
    rmSync(file, { force: true });
```

Reply:
Not agree, extraction rule should be straightforward

Reply:
rollback, but always sceptic